x/proof/types/types.go

```diff
@@ -1,3 +1,58 @@
 package types
 
-// This file is in place to declare the package for dynamically generated protobufs
```

Comment:
Do you mind keeping this comment + empty file and moving the iterator into a `claims_iterator` file?